lib/tasks/va_facilities.rake

```diff
@@ -2,60 +2,60 @@ namespace :va_facilities do
   desc 'Create new VA facility records based on the data from the vamc.json file'
 
   task :create_va_facilities_and_transfer_data => :environment do
-    va_facilities = JSON.parse(File.read("#{Rails.root}/lib/assets/vamc.json"))
+    va_facilities = JSON.parse(File.read("#{Rails.root}/lib/assets/va_facilities.json"))
     visns = Visn.all
-
     visns.each do |visn|
+      puts visn.id
       va_facilities.each do |vaf|
-        if visn.number === vaf["VISN"].to_i && VaFacility.where(station_number: vaf["StationNumber"]).empty?
+        if visn.number === vaf["VISN"].to_i && VaFacility.where(station_number: vaf["Station_Number"]).empty?
           VaFacility.create!(
             visn: visn,
             sta3n: vaf["STA3N"],
-            station_number: vaf["StationNumber"],
-            official_station_name: vaf["OfficialStationName"],
-            common_name: vaf["CommonName"],
-            classification: vaf["Classification"],
-            classification_status: vaf["ClassificationStatus"],
+            station_number: vaf["Station_Number"],
+            official_station_name: vaf["Official_Station_Name"],
+            common_name: vaf["Station_Number"],  #Common_name not supplied in latest VAST file 08_31_2021
+            classification: vaf["CoCClassification"],
+            classification_status: vaf["CoCClassificationAttribute"],
             mobile: vaf["Mobile"],
-            parent_station_number: vaf["ParentStationNumber"],
-            official_parent_station_name: vaf["OfficialParentStationName"],
-            fy17_parent_station_complexity_level: vaf["FY17ParentStationComplexityLevel"],
-            operational_status: vaf["OperationalStatus"],
-            ownership_type: vaf["OwnershipType"],
-            delivery_mechanism: vaf["DeliveryMechanism"],
-            staffing_type: vaf["StaffingType"],
-            va_secretary_10n_approved_date: vaf["VASecretary10NApprovedDate"],
-            planned_activation_date: vaf["PlannedActivationDate"],
-            station_number_suffix_reservation_effective_date: vaf["StationNumberSuffixReservationEffectiveDate"],
-            operational_date: vaf["OperationalDate"],
+            parent_station_number: vaf["Parent_Station_Number"],
+            official_parent_station_name: vaf["Parent_Station_Name"],
+            fy17_parent_station_complexity_level: vaf["Parent_Station_Complexity_Level_2017"],
+            operational_status: vaf["Operational_Status_Active_A_Or_Planned_P_Or_Temporarily_Deactivated_T_Or_Permanently_Deactivated_D"],
+            ownership_type: vaf["Ownership_Type"],
+            delivery_mechanism: vaf["Delivery_Mechanism"],
+            staffing_type: vaf["Staffing_Type"],
+            #va_secretary_10n_approved_date: vaf["VASecretary10NApprovedDate"],
+            #planned_activation_date: vaf["PlannedActivationDate"],
+            #station_number_suffix_reservation_effective_date: vaf["Station_NumberSuffixReservationEffectiveDate"],
```

Comment:
If we don't need these, we can remove them.

Reply:
I removed them.